iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/BranchData.java

```diff
@@ -163,7 +163,7 @@ public BranchData(TwoWindingsTransformer twt, double epsilonX, boolean applyReac
         y = 1 / z;
         ksi = Math.atan2(r, fixedX);
         rho1 = getRho1(twt);
-        rho2 = 1f;
+        rho2 = twt.getTerminal2().getVoltageLevel().getNominalV() / twt.getRatedU2();
```

Comment:
rho2 is the transformer ideal ratio on side 2. 
See http://www.powsybl.org/docs/loadflow/validation.html
I think that rho2 = 1.

Reply:
Corrected.